src/chunk.h

```diff
@@ -985,11 +985,13 @@ static inline bool chunk_is_type(Chunk *pc)
 }
 
 
-static inline bool chunk_is_str(Chunk *pc, const char *str, size_t len)
+static inline bool chunk_is_str(Chunk *pc, const string &str)
```

Comment:
We can keep using "const char *" for the string, so we avoid unnecessary conversion to string objects.
We can determine the length on the string with strlen.

Reply:
Sure. I'll need to adjust the commit message.

Reply:
In hindsight, maybe we should just use a string and get rid of the `memcmp` which is just a goofy C-ism at this point?

Reply:
There is nothing wrong in using C functions.... definitely nothing goofy in it :-) We could perhaps replace ```memcmp``` with strncmp, I missed that on the frist review, but either way would work fine.

Reply:
neovim rocks btw :wink: